### PR TITLE
Case 21454, Case 21448: Fix web interaction and debug print filling logs (RC80)

### PR DESCRIPTION
**Pull request by SamGondelman**
https://highfidelity.manuscript.com/f/cases/21454/RC80-User-cannot-click-or-interact-with-Web-Entities-after-selecting-them-with-Create
https://highfidelity.manuscript.com/f/cases/21448/RC80-Logs-are-constantly-spammed-with-DEBUG-hifi-entities-UNEXPECTED-attempting-to-move-entity

https://github.com/highfidelity/hifi/pull/15040

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17666/HighFidelity-Beta-PR15041-78e9492.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17666/HighFidelity-Quest-Beta-PR15041-78e9492.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17666/HighFidelity-Beta-PR15041-78e9492.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17666/HighFidelity-Beta-Interface-PR15041-78e9492.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17666/HighFidelity-Beta-PR15041-78e9492.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17666/HighFidelity-Beta-Interface-PR15041-78e9492.exe

**Comment by qlimnavarro** [comment]
Currently Testing

File: scripts/system/libraries/entitySelectionTool.js [hunk 1]
```diff
@@ -708,15 +708,15 @@ SelectionDisplay = (function() {
         shape: "Cone",
         solid: true,
         visible: false,
-        ignoreRayIntersection: false,
```
Comment (huffman) [inline scripts/system/libraries/entitySelectionTool.js]: Is ignoreRayIntersection no longer a property?

Comment (SamGondelman) [reply scripts/system/libraries/entitySelectionTool.js]: It's still a property when you're using the Overlays API, but Entities only have `ignorePickIntersection`, so I'm just starting to replace them as I see them.

**Comment by qlimnavarro** [comment]
Pre Merge Fail
Details in 21448

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17683/HighFidelity-Beta-PR15041-b5b004f.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/17683/HighFidelity-Quest-Beta-PR15041-b5b004f.apk)

**Comment by qlimnavarro** [comment]
retesting when built

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17683/HighFidelity-Beta-PR15041-b5b004f.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/17683/HighFidelity-Beta-Interface-PR15041-b5b004f.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17683/HighFidelity-Beta-PR15041-b5b004f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/17683/HighFidelity-Beta-Interface-PR15041-b5b004f.exe

**Comment by qlimnavarro** [comment]
PreMerge Pass

**Pull request merged**